app/models/location.py

```diff
@@ -0,0 +1,26 @@
+from .. import db
+
+
+class Zipcode(db.Model):
+    __tablename__ = 'zip_codes'
+    id = db.Column(db.Integer, primary_key=True)
+    zip_code = db.Column(db.String(10), unique=True)
```

Comment:
You could query with the following (although not sure how inefficient it would be)
```
ZIPCode.query.filter(ZIPCode.zip_code[0:5] == x)
```
Another option would to store the zip and zip+4 separately